settings.py

```diff
@@ -19,7 +19,17 @@
     'debug_toolbar',
 )
 
-MIDDLEWARE = ('debug_toolbar.middleware.DebugToolbarMiddleware',) + MIDDLEWARE
+# If the DebugToolbar be before the GZip, appeared the WARNINGS.
+def insert_debug_toolbar_middleware(middlewares):
+    ret_middleware = list(middlewares)
+
+    for i, middleware in enumerate(ret_middleware):
+        if 'olympia.api.middleware.GZipMiddlewareForAPIOnly' in middleware:
```

Comment:
I mentioned in [my comment](https://github.com/mozilla/addons-server/issues/11819#issuecomment-515924406) to re-use the logic Django-Debug-Toolbar already has for this. It's more code, but it matters because otherwise we're hardcoding the name of the middleware, which would break if we ever change it - in other words it lacks flexibility.